project/Dependencies.scala

```diff
@@ -0,0 +1,50 @@
+import sbt._
+
+/** Dependencies for reuse in different parts of the build */
+object Dependencies {
+
+  // NOTE: remember to change the version numbers in the sample projects
+  // when changing them here
+
+  val scalaVersions = Seq("2.11.12", "2.12.4")
+
+  val slf4j = "org.slf4j" % "slf4j-api" % "1.7.25"
+  val typesafeConfig = "com.typesafe" % "config" % "1.3.2"
+  val reactiveStreamsVersion = "1.0.1"
+  val reactiveStreams = "org.reactivestreams" % "reactive-streams" % reactiveStreamsVersion
+  val reactiveStreamsTCK = "org.reactivestreams" % "reactive-streams-tck" % reactiveStreamsVersion
+
+  def mainDependencies = Seq(slf4j, typesafeConfig, reactiveStreams)
+
+  val junit = Seq(
+    "junit" % "junit-dep" % "4.11",
+    "com.novocode" % "junit-interface" % "0.11"
+  )
+  def scalaTestFor(scalaVersion: String) = {
+    val v = "3.0.4"
+    "org.scalatest" %% "scalatest" % v
+  }
+  val logback = "ch.qos.logback" % "logback-classic" % "1.2.3"
+  val hikariCP = "com.zaxxer" % "HikariCP" % "2.7.4"
+
+  val h2 = "com.h2database" % "h2" % "1.4.191"
```

Comment:
Remember to pick up the version bumps for H2 and postgres since you cut the branch.
And anything else, if that's not all the conflicts on the PR.